task/valint/0.1/valint.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: valint
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Security
+    tekton.dev/tags: CLI, valint
+    tekton.dev/displayName: valint
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    This task can be used to generate sbom and verify the sbom
+  workspaces:
+    - name: source-dir
+      optional: true
+    - name: dockerconfig
+      description: Includes a docker `config.json`
+      optional: true
+      mountPath: /root/.docker
+    - name: output
+      description: Output directory for evidence store.
+  params:
+    - name: args
+      description: extra args that needs to be appended
+      type: array
+    - name: scribe-secret
+      description: Scribe secret name
+      default: scribe-secret
+      type: string
+    - name: version
+      description: valint CLI version run this task
+      default: "latest"
```

Comment:
can we avoid using the latest tag ?

Reply:
@houdini91 can you please take a look at this ?

Reply:
also can you please squash the commits ?

Reply:
* Sorry, missed your comment
* I changed `latest` it to our latest tagged version, (is that enough or do you require it be the repoDigest Hash ref?)
* Squashed and force pushed.